lambdas/README.md

```diff
@@ -1,10 +1,8 @@
 # National Document Repository: Lambdas
 
-This part of the repository contains all our Lambda functions, currently these include...
+[<< Back To README.md](../README.md)
```

Comment:
This now seems a bit of a redundant README. This link just goes back to the README you most probably came from and everything else is just links to other files. Seems to me all this could go on the "main" README